common/djangoapps/third_party_auth/tests/specs/test_lti.py

```diff
@@ -121,7 +121,8 @@ def test_reject_initiating_login(self):
     def test_reject_bad_login(self):
         login_response = self.client.post(
             path=LTI_TPA_LOGIN_URL, content_type=FORM_ENCODED,
-            data="invalid=login"
+            data="oauth_consumer_key=12345&oauth_signature_method=HMAC-SHA1&"
+                 "oauth_signature=STPWUouDw%2FlRGD4giWf8lpGTc54%3D&oauth_timestamp=1436823553&oauth_version=1.0"
```

Comment:
The pre-existing test exists to make sure the server handles invalid input, without crashing the server.  Here, we have changed the test instead of making sure the code handles invalid input.  If this were checked in as is, the server would have 500 errors when users send invalid input.

Rather, you need to update the code to catch the `AttributeError` and leave the test as is.